index.html

```diff
@@ -0,0 +1,146 @@
+				<h2 class="main-title2">2023.07.16(SUN)~22(SAT)</h2>
+				<p>@ Catherine Hall Montego Bay Jamaica.</p>
+			</div>
+		</section>
+		<section id="events" class="main-events">
+			<h2 class="events-title">Events</h2>
+			<hr>
+			<div class="events-cards">
+				<div class="events-card">
+					<img class="events-icon" src="./assets/icons/music.png" alt="">
+					<h3 class="events__card-title">Beach Party</h3>
+					<p class="events-description">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Fuga, provident.</p>
+				</div>
+				<div class="events-card">
+					<img class="events-icon" src="./assets/icons/festival.png" alt="">
+					<h3 class="events__card-title">All white Party</h3>
+					<p class="events-description">Lorem, ipsum dolor, sit amet consectetur adipisicing elit. Corrupti, ducimus doloremque velit.</p>
+				</div>
+				<div class="events-card">
+					<img class="events-icon" src="./assets/icons/concert.png" alt="">
+					<h3 class="events__card-title">Blitz Party</h3>
+					<p class="events-description">Lorem ipsum dolor sit amet, consectetur, adipisicing elit. Qui voluptatem veritatis vitae illo!</p>
+				</div>
+				<div class="events-card">
+					<img class="events-icon" src="./assets/icons/dj.png" alt="">
+					<h3 class="events__card-title">Sound Clash</h3>
+					<p class="events-description">Lorem ipsum dolor sit amet consectetur adipisicing elit. Laboriosam, ad tempora rem ea itaque!</p>
+				</div>
+				<div class="events-card">
+					<img class="events-icon" src="./assets/icons/stage.png" alt="">
+					<h3 class="events__card-title">Main Festival</h3>
+					<p class="events-description">Lorem ipsum dolor sit amet consectetur adipisicing elit. Eos eius quidem corrupti at nesciunt?</p>
+				</div>
+			</div>
+			<a href="" class="program btn">SEE THE WHOLE SCHEDULE
+				</a>
+			<a href="https://www.tixr.com/groups/reggaesumfest/events/reggae-sumfest-2023-62795" class="more-info btn">Book Your tickets</a>
+		</section>
+		<section id="artist" class="featured-artist">
+			<h2 class="featured-title">Featured Artist</h2>
+			<hr>
+				<div id="featured-artist" class="featured-artist-container">
+					
+				</div>
+				<button id="show-more" type='button' class="more btn">More
+					<i class="fa-solid fa-chevron-down"></i>
+				</button>
+		</section>
+		<section id="sponsors" class="main-sponsors">
+			<h2 class="sponsors-title">Sponsors</h2>
+			<hr>
+			<ul class="sponsors-list">
+				<li class="sponsors-item">
+					<img src="./assets/images/redstripe.png" alt=""></li>
+				<li class="sponsors-item">
+					<img src="./assets/images/pepsi.jpeg" alt=""></li>
+				<li class="sponsors-item">
+					<img src="./assets/images/observer.png" alt=""></li>
+				<li class="sponsors-item">
+					<img src="./assets/images/flow.png" alt=""></li>
+				<li class="sponsors-item">
+					<img src="./assets/images/iriefm.jpg" alt=""></li>
+				<li class="sponsors-item">
+					<img src="./assets/images/jtb-logo.jpg" alt=""></li>
```

Comment:
### ADD ALTERNATIVE TEXT TO ALL YOUR IMAGES WITHOUT ANY 

- Hey, let's add an alternative text to ALL THE IMAGES without the alt text in the HOME and ABOUT PAGE  to improve the web accessibility of your webpage.